wiki/People/The_Team/osu!_Alumni/en.md

```diff
@@ -113,6 +113,7 @@ The [osu! Alumni group page](https://osu.ppy.sh/groups/16) lists all of the memb
 | ![][flag_KR] [KRZY](https://osu.ppy.sh/users/114017) | Chat Moderator |
 | ![][flag_JP] [KSHR](https://osu.ppy.sh/users/409957) | BAT, GMT, QAT |
 | ![][flag_HU] [Kurokami](https://osu.ppy.sh/users/260933) | QAT |
+| ![][flag_BR] [Kuron-kun](https://osu.ppy.sh/users/2697284) | QAT, GMT |
```

Comment:
```suggestion
| ![][flag_BR] [Kuron-kun](https://osu.ppy.sh/users/2697284) | GMT, QAT |
```
these are alphabetical rather than chronological, for now